<!DOCTYPE HTML>
<!--
	Hielo by TEMPLATED
	templated.co @templatedco
	Released for free under the Creative Commons Attribution 3.0 license (templated.co/license)
-->
<html>
	<head>
		<title>ARDVARC</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="subpage">

		<!-- Header -->
			<header id="header">
				<div class="logo"><a href="index.html">Antimicrobial Resistance Data Visualization And Rapid Classification</a></div>
				<a href="#menu">Menu</a>
			</header>

		<!-- Nav -->
			<nav id="menu">
				<ul class="links">
					<li><a href="index.html">Home</a></li>
					<li><a href="analyze.html">Analyze</a></li>
					<li><a href="about.html">About</a></li>
				</ul>
			</nav>

		<!-- One -->
			<section id="One" class="wrapper style3">
				<div class="inner">
					<header class="align-center">
						<p>Advancing the Field in the Lab</p>

						<h2>About ARDVARC</h2>
					</header>
				</div>
			</section>

		<!-- Two -->
				<div id="main" class="container">

							<h2>Why Antimicrobial Resistance?</h2>
							<p></span>Antimicrobial resistance (AMR) is a growing threat. Researchers and medical professionals have been struggling to gather and centralize data on antibiotic-resistant pathogens, and only recently have more comprehensive databases for tracking of AMR infections and genes emerged. </p>
							<h2>What is ARDVARC?</h2>
							<p><span class="image left"><img src="images/megaplate.png" alt></span><p>ARDVARC builds on the ease of culturing and sequencing bacteria that have evolved in lab settings to produce AMR-related mutations. By comparing a wild-type genome against a large set of possible past or future mutations, we can point to places where future mutations might occur for this genome. This makes it easier to find AMR-related patterns in the genome. Given continuously falling genome sequencing prices and the subsequent rise in accessibility of whole-genome sequencing, we aim to consolidate the computational work involved in processing the sequenced genomes. With ARDVARC, researchers can conveniently perform almost all the bioinformatics analysis they need in one easy step.</p>
							<br/>

							<h2>Who is ARDVARC for?</h2>
							<p>Researchers in hospitals and universities will find ARDVARC useful as they collect samples of wild-type bacteria and seek to understand what AMR-related mutations might occur next.</p>

							<h2>What else could ARDVARC be used for?</h2>
							<p>When complete, we anticipate that ARDVARC can be used for comparing genomes in a variety of applications - anywhere where high-throughput culturing and the resulting sequencing data is available. ARDVARC can be adapted to check a genome against past and future mutations for any purpose, not just AMR.</p>

							<h2>How could ARDVARC be improved?</h2>
							<p>Right now, ARDVARC needs the following:</p>
							<ul>
								<li>A database, and tagging/metadata functions. It is not currently connected to a database, but this would be a vital step in availability and functionality.</li>
								<li>More computational power. In a non-demo version, ARDVARC takes a very long time to run.</li>
								<li>More data. While some genomic data is available, the data that's useful for ARDVARC must be of a specific type.</li>
							</ul>

							<h2>Who is THE RESISTANCE?</h2>
							<p><span class="image left"><img src="images/team.jpg" alt></span><p>We are a diverse group of biologists, engineers, and computer scientists all with the shared goal of improving the treatment of the antibiotic resistance crisis. <strong>Rachel Diao, Cassidy Mileti</strong>, and <strong>Martin St√ºbler</strong> make up the the biological sciences division, responsible for understanding the problems and needs of researchers in the field. <strong>Julian Eng</strong> and <strong>Suzie Petryk</strong> are data scientists who created a random-forest-based algorithm to process genomic data. <strong>Jessie Taft</strong> is responsible for the the visualization of the data and website design. </p>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
<br/>
							<h2>How did you make this project?</h2>
							<h3>Genome Classification</h3>
							<p>The genome classifier was written in Python. It uses a random forest algorithm to identify sites that contribute most to variation in resistance.</p>
							<h3>Visualization & Web</h3>
							<p>The visualization tool was adapted from work by <a href="https://github.com/lairdm/islandplot">Matthew Laird</a>.</p>
							<h3>Sponsorship & Support</h3>
							<p>ARDVARC was created for the 2018 Health Hackathon, sponsored by Cornell University, the National Science Foundation, GlaxoSmithKline, and Johnson & Johnson. The team thanks the Hackathon mentors for their ideas and encouragement, especially David Orbach, Ilana Brito, and Laura Goodman.</p>

				</div>

		<!-- Footer -->
			<footer id="footer">
				<div class="copyright">
					Made by THE RESISTANCE using the Hielo template from <a href="https://templated.co/hielo">templated.co</a>
				</div>
			</footer>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
